apps/vscode-extension/src/intellisense/design-token.ts

```diff
@@ -10,9 +10,45 @@ type DesignTokenCache = {
 
 const readFile = async (filePat: string) => (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat))).toString();
 
+const getFilesPatternsFromProjectConfiguration = async (currentFile: string): Promise<string[] | undefined> => {
```

Comment:
How long does the execution take on a big project ? Won't we risk to slow down the IDE ?